Jenkinsfile

```diff
@@ -1 +1 @@
-elifeFormula('elife-metrics')
+elifeFormula('elife-metrics', '/srv/elife-metrics', 'elife-metrics-formula', ['s1604'])
```

Comment:
same as seen in `journal`, the 3rd argument can be simplified to `null` as it makes the environment names shorter:
https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/elifeFormula.groovy#L11-L20